Strategies/protocols/psyfi/actions/deposit.ts

```diff
@@ -52,6 +61,45 @@ export const deposit = async (
     vaultOwnershipAccount = address
   }
 
+  const poolMintATA = await Token.getAssociatedTokenAddress(
+    ASSOCIATED_TOKEN_PROGRAM_ID,
+    TOKEN_PROGRAM_ID,
+    NATIVE_MINT,
```

Comment:
is this meant to be for 1) all tokens or 2) only wSOL token? relevant collateral asset mint address should be passed here 1)

Reply:
treasuryAssetAccount.isSol ? NATIVE_MINT: form.strategy.vaultAccounts.lpTokenMint,